website/docs/plugins/nevergrad_sweeper.md

```diff
@@ -112,23 +113,42 @@ name: nevergrad
 
 ## Defining the parameters
 
-The plugin can use 2 types of parameters:
+The plugin supports 2 types of parameters: [Choices](https://facebookresearch.github.io/nevergrad/parametrization_ref.html#nevergrad.p.Choice) and [Scalars](https://facebookresearch.github.io/nevergrad/parametrization_ref.html#nevergrad.p.Scalar). They can be defined either through config file or commandline override.
```

Comment:
It's not the plugin, it's Nevergrad, right?

Reply:
Here's the list of params nevergrad supports :https://facebookresearch.github.io/nevergrad/parametrization_ref.html#parameters

Our plugins covers half of them. For future improvements we could look into support more :)

Reply:
roger.